Sprint-2/implement/contains.js

```diff
@@ -1,3 +1,11 @@
-function contains() {}
+function contains(someObject, propertyName) {
+  let whatTheRobotFound = someObject[propertyName];
+  if (someObject[propertyName] === undefined) {
```

Comment:
We can store the value `undefined` in objects. What would you expect to be returned in this case? What does your code do?

```javascript
contains({"name": undefined}, "name");